common/flatpak-dir.c

```diff
@@ -11893,15 +11893,60 @@ flatpak_dir_find_local_related (FlatpakDir   *self,
 }
 
 static GDBusProxy *
-get_accounts_dbus_proxy (void)
+get_localed_dbus_proxy (void)
+{
+  const char *accounts_bus_name = "org.freedesktop.locale1";
+  const char *accounts_object_path = "/org/freedesktop/locale1";
+  const char *accounts_interface_name = accounts_bus_name;
+
+  return g_dbus_proxy_new_for_bus_sync (G_BUS_TYPE_SYSTEM,
+                                        G_DBUS_PROXY_FLAGS_NONE,
+                                        NULL,
+                                        accounts_bus_name,
+                                        accounts_object_path,
+                                        accounts_interface_name,
+                                        NULL,
+                                        NULL);
+}
+
+static char *
+get_localed_default_locale (void)
 {
-  g_autoptr(GDBusConnection) conn = NULL;
+  GVariant *locale_variant;
+  g_autoptr(GDBusProxy) proxy = NULL;
+
+  proxy = get_localed_dbus_proxy ();
+  if (proxy == NULL)
+    return NULL;
+
+  locale_variant = g_dbus_proxy_get_cached_property (proxy, "Locale");
```

Comment:
`locale_variant` is leaked.

Reply:
Ah, yeah it turns out copying code from gnome-control-center and assuming it doesn't have leaks doesn't turn out well. Looks like the leaks have already been fixed upstream (I was copying from the Endless fork).